src/ui/components/pages/assets/assetItem.styl

```diff
@@ -11,15 +11,77 @@
   position: relative;
   z-index: 1;
   margin-bottom: 8px;
-  cursor: pointer;
+  cursor: default;
```

Comment:
It's better to remove `cursor` prop here. With `cursor: default` it looks like you can't select asset name or balance, because it always looks like an arrow. Without this prop cursor will change to a vertical line when it's above the text.